### PR TITLE
fix(table): render download links for file questions in tables

### PR DESCRIPTION
**Pull request by czosel**
I left out the extra query to the caluma backend for the moment. We can add it later if it turns out to be needed.

### CODE REVIEW

File: addon/components/cf-field-value.js [hunk 1]
```diff
@@ -1,34 +1,63 @@
...
         const answerValue = field.answer.value || [];
         const options = field.question.multipleChoiceOptions.edges.filter(
           edge => answerValue.includes(edge.node.slug)
         );
-        return options && options.length
-          ? options.map(edge => edge.node.label).join(", ")
-          : answerValue.join(", ");
+        return {
+          label:
+            options && options.length
```
Comment (fkm) [inline addon/components/cf-field-value.js]: Is it possible that there are no options but a non-empty answerValue?

Comment (czosel) [reply addon/components/cf-field-value.js]: Not sure. I didn't change anything here, this was just prettier doing some reformatting. Let's leave it for the moment :wink:

Comment (fkm) [reply addon/components/cf-field-value.js]: Okey :-)

Plus. I just realized that the chosen options could have been removed from the question.

Comment (czosel) [reply addon/components/cf-field-value.js]: :+1: Is this good to go, then?

**Pull request merged**